joblib/_parallel_backends.py

```diff
@@ -301,6 +304,8 @@ class ThreadingBackend(PoolManagerMixin, ParallelBackendBase):
     """
 
     supports_timeout = True
+    use_threads = True
```

Comment:
Slight preference for `uses_threads` instead. `use_threads` feels a bit off grammatically, especially given other notations like `supports_timeout`.